Imagekit/Helper/TransformationTypes.cs

```diff
@@ -407,6 +218,18 @@ public Transformation EffectGray()
         return this.Add("e-grayscale", "true");
     }
 
+    /// <summary></summary>
+    public Transformation EffectShadow()
+    {
+        return this.Add("e-shadow", "true");
+    }
+
+    /// <summary></summary>
+    public Transformation EffectGradient()
+    {
+        return this.Add("e-gradient", "true");
```

Comment:
There is no such transformation as e-gradient-true, e-shadow has a syntax like this `e-gradient-<linear-direction>_<from-color>_<to-color>_<stop-point>` or just e-gradient
https://imagekit.io/docs/effects-and-enhancements#gradient---e-gradient

Reply:
Fixed it